Set channel_targets for dev release and re-render feedstock

aakashmandavilli96 opened the pull request:
Rerendered the feedstock locally and published the PR by running the `conda smithy rerender`

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```, ```recipe/meta.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint --conda-forge .` from the recipe directory. You can also examine the [workflow logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18806135670) for more detail.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18806135670](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18806135670). Examine the logs at this URL for more detail.</sub>

aws-prayags commented:
@aakashmandavilli96 this is for a pre-release build right? can you link the code change in this branch showing that the label is [correctly set](https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build) for pre-release build ?

aakashmandavilli96 committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.27.15.27.40

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.9

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

aakashmandavilli96 commented:
> @aakashmandavilli96 this is for a pre-release build right? can you link the code change in this branch showing that the label is [correctly set](https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build) for pre-release build ?

@aws-prayags I have added the following in the `conda_build_config.yaml` which is responsible for the pre-release build, and also ran rerender based on the following [suggestion](https://github.com/conda-forge/sagemaker-code-editor-feedstock/pull/192#pullrequestreview-3321616910)

```
channel_targets:
  - conda-forge sagemaker-code-editor_dev
```

aws-prayags commented:
> > @aakashmandavilli96 this is for a pre-release build right? can you link the code change in this branch showing that the label is [correctly set](https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build) for pre-release build ?
> 
> @aws-prayags I have added the following in the `conda_build_config.yaml` which is responsible for the pre-release build, and also ran rerender based on the following [suggestion](https://github.com/conda-forge/sagemaker-code-editor-feedstock/pull/192#pullrequestreview-3321616910)
> 
> ```
> channel_targets:
>   - conda-forge sagemaker-code-editor_dev
> ```

@aakashmandavilli96 can we make the `conda_build_config.yaml` change in a separate PR and merge that first to this branch ? let's make sure that branch is correctly setup before we release any new package from it

aakashmandavilli96 commented:
> > > @aakashmandavilli96 this is for a pre-release build right? can you link the code change in this branch showing that the label is [correctly set](https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build) for pre-release build ?
> > 
> > 
> > @aws-prayags I have added the following in the `conda_build_config.yaml` which is responsible for the pre-release build, and also ran rerender based on the following [suggestion](https://github.com/conda-forge/sagemaker-code-editor-feedstock/pull/192#pullrequestreview-3321616910)
> > ```
> > channel_targets:
> >   - conda-forge sagemaker-code-editor_dev
> > ```
> 
> @aakashmandavilli96 can we make the `conda_build_config.yaml` change in a separate PR and merge that first to this branch ? let's make sure that branch is correctly setup before we release any new package from it

Yes can make it if suggested but this is not releasing any new version for now right, please let me know  if we see concerns here can still have this change on a other PR

aws-prayags commented:
@aakashmandavilli96 I see `Bumped the build number (if the version is unchanged)` is checked - so I assumed we're releasing a pre-release version as part of this PR. Please update desc if so.

arkaprava08 reviewed (approved):
LGTM

aakashmandavilli96 commented:
> @aakashmandavilli96 I see `Bumped the build number (if the version is unchanged)` is checked - so I assumed we're releasing a pre-release version as part of this PR. Please update desc if so.

Based on offline conversation removed the `conda_build_config.yaml` for now and will add in the next PR

aws-prayags commented:
This PR is to set up the `1.7.0dev` branch for pre-releases to a dev label. It also re-renders feedstock for the updated label to apply. Approving.
